client/Dockerfile

```diff
@@ -1,19 +1,5 @@
-FROM debian:latest
+FROM node:0.10
```

Comment:
That was alpine-node; the node image is something we only recently learned about.  Its what we use in service, and updating to use it here makes the builds significantly faster.  Whats more, the node image is just debian + node: https://github.com/joyent/docker-node/blob/787d816f90dc1bf933b6503b876bdeb752b24f96/0.10/Dockerfile